demos/guessing_game.py

```diff
@@ -1,6 +1,9 @@
 import asyncio
 import random
 from pathlib import Path
+import sys
+
+sys.path.append(str(Path(__file__).resolve().parent / 'src'))
```

Comment:
Instead, right-click the `src` folder in PyCharm -> Mark Directory As -> Sources Root.
You might also need to do the same for the Quest folder. 

Then remove this code.